docker/packager/centos/Dockerfile

```diff
@@ -0,0 +1,64 @@
+FROM centos:7
+# Usage:
+# docker build -t yandex/centos-builder .
+# docker run --rm --volume $(pwd)/../../..:/build/  --volume /tmp:/output -it yandex/centos-builder
+
+# # set timezone
+# RUN rm /etc/localtime
+# RUN ln -s /usr/share/zoneinfo/Europe/Moscow /etc/localtime
+
+
+RUN yum -y install epel-release centos-release-scl \
+    # https://bugs.centos.org/view.php?id=14773
+    && sed '/\[centos-sclo-sclo-source\]/,/gpgkey/d' -i /etc/yum.repos.d/CentOS-SCLo-scl.repo \
+    && sed '/\[centos-sclo-rh-source\]/,/gpgkey/d' -i /etc/yum.repos.d/CentOS-SCLo-scl-rh.repo \
+    && yum -y install \
+        rpmdevtools sudo rpmlint rpm-build redhat-rpm-config createrepo wget git python \
+    && wget --no-check-certificate https://raw.githubusercontent.com/Kentzo/git-archive-all/master/git_archive_all.py \
```

Comment:
Running the script from "master" branch is pretty unsafe if we will put it in our CI.